src/xyz.cpp

```diff
@@ -0,0 +1,139 @@
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n4");
+  //   fprintf(outf, "\t%.8f", (0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", 0.);
+  //   fprintf(outf, "\t%.8f", -fib.mol.diameterAtom);
+  // }
+  //
+  //
+  //
+  //
+  for (int i = 0; i < fib.mol.numAtoms; i++) {
+    fprintf(outf, "\n5");
+    fprintf(outf, "\t%.8f", (box + i * fib.mol.distanceAtoms) / res);
+    fprintf(outf, "\t%.8f", 0.);
+    fprintf(outf, "\t%.8f", 0.);
+  }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n2");
+  //   fprintf(outf, "\t%.8f", (box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", fib.mol.diameterAtom);
+  //   fprintf(outf, "\t%.8f", 0.);
+  // }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n2");
+  //   fprintf(outf, "\t%.8f", (box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", -fib.mol.diameterAtom);
+  //   fprintf(outf, "\t%.8f", 0.);
+  // }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n4");
+  //   fprintf(outf, "\t%.8f", (box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", 0.);
+  //   fprintf(outf, "\t%.8f", fib.mol.diameterAtom);
+  // }
+  // //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n4");
+  //   fprintf(outf, "\t%.8f", (box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", 0.);
+  //   fprintf(outf, "\t%.8f", -fib.mol.diameterAtom);
+  // }
+  //
+  //
+  //
+  //
+  //
+  for (int i = 0; i < fib.mol.numAtoms; i++) {
+    fprintf(outf, "\n5");
+    fprintf(outf, "\t%.8f", (-box + i * fib.mol.distanceAtoms) / res);
+    fprintf(outf, "\t%.8f", 0.);
+    fprintf(outf, "\t%.8f", 0.);
+  }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n2");
+  //   fprintf(outf, "\t%.8f", (-box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", fib.mol.diameterAtom);
+  //   fprintf(outf, "\t%.8f", 0.);
+  // }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n2");
+  //   fprintf(outf, "\t%.8f", (-box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", -fib.mol.diameterAtom);
+  //   fprintf(outf, "\t%.8f", 0.);
+  // }
+  //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n4");
+  //   fprintf(outf, "\t%.8f", (-box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", 0.);
+  //   fprintf(outf, "\t%.8f", fib.mol.diameterAtom);
+  // }
+  // //
+  // for (int i = 0; i < fib.mol.numAtoms; i++) {
+  //   fprintf(outf, "\n4");
+  //   fprintf(outf, "\t%.8f", (-box + 0 * fib.offset + i * fib.mol.distanceAtoms) / res);
+  //   fprintf(outf, "\t%.8f", 0.);
+  //   fprintf(outf, "\t%.8f", -fib.mol.diameterAtom);
+  // }
+
```

Comment:
Are all the commented lines left in for future? Also there's a lot of repeated printing things like "\n4" and "%t.8f",0. could this be handled in a "printtofile" function to save on repeated lines? Not urgent.

Reply:
I'm not even sure this file is still needed. Will probably delete it in on one of the next commits.

Reply:
This was a quick  thing to create snapshots for Andela that she requested last minute a few weeks ago for her talk

Reply:
Removed it. Live in the newest commit.